components/admin_console/plugin_management/plugin_management.test.jsx

```diff
@@ -405,6 +411,7 @@ describe('components/PluginManagement', () => {
             config: {
                 ...defaultProps.config,
                 PluginSettings: {
+                    ...defaultProps.config.PluginSettings,
```

Comment:
Should we add a variant with `Enable: false` that verifies the marketplace option is, itself, disabled?

Reply:
not gonna like this but that case is already tested through the new snapshot :P (https://github.com/mattermost/mattermost-webapp/blob/1177623294319de1e109d43c76ba42469050b467/components/admin_console/plugin_management/plugin_management.test.jsx#L109)

Reply:
Ah, snapshots. Grumble, grumble. Fine to defer, but these should die a painful death :D

Reply:
hehe, i know! i'll be the first with the 🔪 to end these snapshots whenever we refactor the plugin_management component.